toxcore/group.c

```diff
@@ -878,13 +878,23 @@ static void rejoin_frozen_friend(Group_Chats *g_c, int friendcon_id)
     }
 }
 
+static int g_handle_any_status(void *object, int friendcon_id, uint8_t status, void *userdata)
+{
+    Group_Chats *g_c = (Group_Chats *)object;
+
+    if (status) {
```

Comment:
`if (status != ...something?)` some constant here? I think this is an enumeration in practice.

Reply:
It's not an enum, just 0 or 1. Presumably we'll make it a bool one day. So "(status)" seems correct.